proto/frequenz/api/microgrid/v1/microgrid.proto

```diff
@@ -507,6 +538,27 @@ enum ComponentBoundsTargetMetric {
   COMPONENT_BOUNDS_TARGET_METRIC_POWER_REACTIVE = 6;
 }
 
+// Request parameters for the RPC `AddComponentBounds`.
+message AddComponentBoundsRequest {
+  // The ID of the target component.
+  uint64 component_id = 1;
+
+  // The target metric whose bounds have to be set.
+  ComponentBoundsTargetMetric target_metric = 2;
+
+  // A list of bounds for the target metric.
+  // While incorporating these bounds, the API will ensure that overlapping
+  // pairs of bounds are coalesced into a single pair of bounds, and disjointed
```

Comment:
merged and non-overlapping might be better terms?

Reply:
Done